### ISSUE TITLE
Custom Netlify configuration properties

### ISSUE THREAD

**Issue by 8eecf0d2** [8eecf0d2/netlify-local#12]
In addition to #11, the Netlify configuration could also be used for other **netlify-local** options.

```toml
[plugins.local.webpack]
  config = "webpack.js"
[plugins.local.server]
  static = true
  lambda = true
  certificates = "private/certs"
  port = 9000
[plugins.local.functions]
  source = "src/lambda/ts"
  files = [
    "foo.js",
    "nested/bar.js"
  ]
```

This isn't as pretty as I'd like but I'm new to toml so maybe there's a better way to write this

**Comment by jayvdb** [comment]
A lot of this is already done.  It just needs some documenting.

**Comment by 8eecf0d2** [comment]
@jayvdb yeah, I was waiting for Netlify to provide guidance on modifying the `netlify.toml` with plugin stuff for a while but implemented most of this and recently merged in into master.

I have some [docs in the wiki](https://github.com/8eecf0d2/netlify-local/wiki/Serve-Options) but will refactor to 3 or so pages like **Options**, **Serve**, **Build**.

**Comment by 8eecf0d2** [comment]
Closing this as it's mostly implemented and prefer #11 for bundle feature and related discussions.
